### DISCUSSION TITLE
How to edit placeholders for InputPicker

### DISCUSSION THREAD

**Discussion by fuzunspm** [rsuite/rsuite#1802] [Q&A] [answered]
Hi,

Other than main placeholder text I want to change "no results found" and "Create option xxxx" texts but couldn't find how to do that.

Thanks in advance

**Comment by superman66** [comment, marked as answer]
@fuzunspm  you can use `locale` to custom your text.
```jsx
      <InputPicker
        data={[]}
        creatable
        locale={{
          newItem: "Create New Item",
          noResultsText: "no results found",
          createOption: 'Create option "{0}"'
        }}
      />
```

**Reply by fuzunspm** [reply]
awesome! thank you very much

**Reply by fuzunspm** [reply]
in addition to that, how can i check all of possible keys for locale?

**Reply by superman66** [reply]
For now, you can only find it here: https://github.com/rsuite/rsuite/blob/next/src/locales/en_US.ts
We will improve the documentation in the v5 release.
